### PR TITLE
Remove Hardcoded values from migration file

### PR DESCRIPTION
**Pull request by muhammad-ali-e**
## What

- Substituted hardcoded values within migrations with relevant environment variables.

## Note for Existing Installations [Please Add to Release note]:

- Your admin username remains as `admin` and the password remains `ascon`.
- To update the password, we **highly recommend** using the Django Admin Panel:
  - Access the admin page via `/admin` and log in.
  - Change the password from there.
- Alternatively, as an option of last resort, you can perform a manual Database Update:
  -  Password and username can be directly modified in the DB table `public.user`. However, this method is **not preferred** due to potential risks and complexities.

## Why

- Hardcoded values are not recommended for security and flexibility reasons.

## How

- Accepting values from ENV

## Env Config

- `SYSTEM_ADMIN_USER`: (optional) Specify the system admin username.
- `SYSTEM_ADMIN_PASSWORD`: (optional) Specify the system admin password.

## Notes on Testing

- Migrations should function correctly.
- Accessing the admin page without enabling it (ADMIN_ENABLED set to False) should result in a 404 error.

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/account/migrations/0002_auto_20230718_1040.py [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 # Generated by Django 4.2.1 on 2023-07-18 10:40
 
+from django.conf import settings
 from django.contrib.auth.hashers import make_password
 from django.db import migrations
 
 
 def create_public_tenant_and_domain(apps, schema_editor):
-    Organization = apps.get_model("account", "Organization")
+    organization_model = apps.get_model("account", "Organization")
```
Comment (chandrasekharan-zipstack) [inline backend/account/migrations/0002_auto_20230718_1040.py:10]: NIT: @muhammad-ali-e why have we renamed it here? We don't seem to follow this for other data migrations

Comment (muhammad-ali-e) [reply backend/account/migrations/0002_auto_20230718_1040.py:10]: Sonar issue raised : variable name  should follow the snakecase

Comment (chandrasekharan-zipstack) [reply backend/account/migrations/0002_auto_20230718_1040.py:10]: Ah got it, maybe we could try importing the models within this function. That way we can probably type it too. We might need `from __future__ import annotations` alone

**Review by hari-kuriakose — APPROVED** [review]
@muhammad-ali-e LGTM, provided we have to edit existing migration file in this case, which is usually not recommended.

However, agree and let's do add your [notes regarding existing installations](https://github.com/Zipstack/unstract/pull/214#issue-2217591280) to the release notes without fail.

**Pull request merged**